src/CoinbaseSmartWalletFactory.sol

```diff
@@ -5,22 +5,36 @@ import {LibClone} from "solady/utils/LibClone.sol";
 import {CoinbaseSmartWallet} from "./CoinbaseSmartWallet.sol";
 
 /// @title Coinbase Smart Wallet Factory
+///
 /// @notice CoinbaseSmartWallet factory, based on Solady's ERC4337Factory.
+///
 /// @author Coinbase (https://github.com/coinbase/smart-wallet)
 /// @author Solady (https://github.com/vectorized/solady/blob/main/src/accounts/ERC4337Factory.sol)
 contract CoinbaseSmartWalletFactory {
-    /// @dev Address of the ERC4337 implementation.
+    /// @notice Address of the ERC-4337 implementation used to deploy new cloned accounts.
     address public immutable implementation;
 
+    /// @notice Reverted when trying to create a new `CoinbaseSmartWallet` account without any owner.
```

Comment:
I think I prefer the verb `Thrown` when describing errors.

Reply:
I tend to prefer 'reverts' as it is literally what we write (i.e., "revert MyError();") but open to debate. Maybe @wilsoncusack could share his view as well?